sonar-client/go.mod

```diff
@@ -5,5 +5,5 @@ go 1.13
 require (
 	github.com/stretchr/testify v1.7.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/apimachinery v0.18.9
+	k8s.io/apimachinery v0.17.4
```

Comment:
@tylergu 
Can you confirm these two things:
1. since you are not using the operator-sdk to build the image, can you check whether `k8s.io/apimachinery v0.18.9` works when building the yugabyte image? I am not sure whether the previous build failure is caused by operator-sdk or not. If `k8s.io/apimachinery v0.18.9` works we do not need to change the version
2. if the yugabyte has to use v0.17.4 anyway, can you verify whether this version also works with all other controllers we have? Can we still build all other controllers successfully with v0.17.4? I remember previously I encountered a problem when building one of the controllers using older version `k8s.io/apimachinery` so later I change to v0.18.9.

Reply:
@tylergu You can make it a special case in the build.py script: if the operator is the yugabyte one you change the version to `v0.17.4`, otherwise you keep it as `v0.18.9` as before. So that we do not need to worry about whether this change breaks other controllers

Reply:
For some reason, simply adding a replace statement at the end of the go.mod file does not fix this. The commit reads the go.mod file after the sonar-client being copied into the dep-sonar and modify the line inplace.